purchase_batch_invoicing/wizards/purchase_batch_invoicing.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from openerp import _, api, fields, models
+from openerp.exceptions import UserError
+
+
+class WizardModel(models.TransientModel):
+    _name = "purchase.batch_invoicing"
+
+    purchase_order_ids = fields.Many2many(
+        comodel_name="purchase.order",
+        string="Purchase orders",
+        domain="[('invoice_status', '=', 'to invoice')]",
+        required=True,
+        readonly=True,
+        default=lambda self: self._default_purchase_order_ids(),
+    )
+    grouping = fields.Selection(
+        selection=[
+            ("id", "Purchase Order"),
```

Comment:
I know but it helps maintainance in the future and code readibility.
Non-blocking for me